package.json

```diff
@@ -46,12 +46,12 @@
     "@types/compression": "^1.7.0",
     "@types/cors": "^2.8.10",
     "@types/express": "^4.17.11",
-    "@types/jest": "^26.0.22",
+    "@types/jest": "^26.0.23",
     "@types/morgan": "^1.9.2",
-    "@types/node": "^14.14.37",
-    "@types/node-fetch": "^2.5.8",
-    "@types/split2": "^2.1.6",
+    "@types/node": "^15.0.2",
```

Comment:
But we're using node 14 🤔

Reply:
More reason to move to 16.

Reply:
Sent https://github.com/w3c/respec-web-services/pull/197